Jenkinsfile

```diff
@@ -385,12 +387,18 @@ def nodeWithCleanup(String label, Closure steps) {
   }
 }
 
-def xbuildSafe(String command) {
+def xbuild(String arguments) {
   try {
-    sh "${command}"
+    sh "${xbuildCmd} ${arguments} 1> xbuildOutput"
+    def stdOut = readFile 'xbuildOutput'
+
+    echo stdOut
   } catch (err) {
-    if (err.getMessage().contains("Assertion at gc.c:910, condition `ret != WAIT_TIMEOUT' not met")) {
-      echo "StyleCop crashed. No big deal."
+    def stdOut = readFile 'xbuildOutput'
+    echo stdOut
+
+    if (stdOut.contains("Assertion at gc.c:910, condition `ret != WAIT_TIMEOUT' not met")) {
```

Comment:
There is a single quote in the string - I thought using double quotes looks better than escaping it, but I can switch to single quotes if that's the preferred way.